README.md

```diff
@@ -55,4 +61,24 @@ With these values the following formula is used to calculate the `minReplicas` v
 minReplicas = Ceiling ( delta + ( resultFromQuery / requestsPerReplica ) )
 ```
 
-By tuning the `delta` and `requestsPerReplica` values it should be possible to follow the curve of the number of requests coming out of the Prometheus query closely and stay just below the number of replicas that the `HorizontalPodAutoscaler` would come up with under normal circumstances. If the curve is higher you're wasting resources, if it's much lower than it provides less safety.
\ No newline at end of file
+By tuning the `delta` and `requestsPerReplica` values it should be possible to follow the curve of the number of requests coming out of the Prometheus query closely and stay just below the number of replicas that the `HorizontalPodAutoscaler` would come up with under normal circumstances. If the curve is higher you're wasting resources, if it's much lower than it provides less safety.
+
+### Limit the rate of scale down
+
+It can cause problems that the built in horizontal pod auto scaler can scale down a service too quickly if the CPU load drops. There is no built-in way to limit how big portion of the current pod count the auto scaler can remove in one step.  
```

Comment:
Perhaps explain what a 'step' is as well and how it's globally configured for this controller and has some randomness to it's interval.

Reply:
I've added some info about how the HPA works.  
Do you remember where did you see info about the randomness to its interval? [Here](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) I found that you can control the interval with the flag `--horizontal-pod-autoscaler-sync-period`, but I couldn't find any info about it being random.

Reply:
Hm, it's actually hardcoded here: https://github.com/estafette/estafette-k8s-hpa-scaler/blob/master/main.go#L188-L190

Reply:
Oh, you mean the interval of `estafette-k8s-hpa-scaler` itself? I thought you meant the interval of the Horizontal Pod Autoscaler ^^

Reply:
The interval of `estafette-k8s-hpa-scaler` determines how frequently the hpa gets evaluated and hence affects the speed at which it scales down if you use the ratio approach.

Reply:
Although in your graph it somehow looks like 5 minute intervals. Curious why that is. Perhaps it takes a single run quite some time to check all HPAs, but it seems unlikely with only a couple of them having the required annotation.

Reply:
I think that's due to the default [cooldown delay](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-cooldown-delay), which controls how much the HPA has to wait between subsequent scale down operations. Its default is 5 minutes, so that matches what I saw when I tested it.